web/src/features/fbaCalculator/components/FBATable.tsx

```diff
@@ -380,7 +401,10 @@ const FBATable = (props: FBAInputGridProps) => {
             variant="contained"
             color="primary"
             spinnercolor="white"
-            onClick={deleteSelectedStations}
+            onClick={() => {
+              deleteSelectedStations()
+              checkTable()
```

Comment:
An easier, more reactive way would be to conditionally render the table if rows are empty, or the table instructions otherwise: https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator

I can show you what I mean if you like.

Reply:
Ah yes, Sy mentioned there might be a better way to do this. I will read up and attempt it myself, if I hit a block I will reach out. Thanks Connor!